src/components/AppointmentResultMain.jsx

```diff
@@ -1,11 +1,97 @@
-import React from "react";
+import React, {useState} from "react";
+import PageLayout from "../components/PageLayout";
+import Header from "../components/Header";
+import Footer from "../components/Footer";
+import api from "../http/api";
+import {toast, ToastContainer} from "react-toastify";
 
 function AppointmentResultMain() {
-    return(
-        <>
-            REZULTATI
-        </>
+
+    const [therapyResultData , setTherapyResultData] = useState({
+        therapyId: '',
+        result: '',
+        status: '',
+    });
+
+    const handleChange = (e) => {
+        const {name, value} = e.target;
+        setTherapyResultData({...therapyResultData, [name]: value});
+    }
+
+    const [selectedOption, setSelectedOption] = useState(null);
+    const handleOptionChange = (event) => {
+        setSelectedOption(event.target.value);
+    };
+
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+        await api.post("/therapy/result", therapyResultData, {
+            headers: {
+                'Authorization': 'Bearer ' + localStorage.getItem('token')
+            }
+        }).then(() => {
+            toast.success("Uspjesno unesen rezultat terapije")
+        }).catch(() => toast.error("Dogodila se greška."));
+    }
+
+    return (
+        <PageLayout>
+            <div className={'flex flex-col h-full items-center justify-center text-sky-900 font-bold mx-3 text-center'}>
+                <form onSubmit={handleSubmit}
+                      className={'w-full md:w-5/6 xl:w-1/2 flex flex-col justify-center bg-sky-200 p-10 h-full my-5 rounded-lg'}>
+                    <h1 className={'font-bold mb-8 text-3xl'}>Rezultati ciklusa terapije</h1>
+                    <div style={{ display: 'flex'}}>
+                        <label className={'text-start p-2 mr-10'}>Ime i prezime pacijenta: </label>
+                        <label className={'text-start p-2 text-black font-medium'}>Luka vivoda</label>

```

Comment:
Instead of place holder insert real data. Fetch therapy and patient informations and all required data and display it.

Reply:
What are the right url's for these fetches? Can't seem to find them